Makefile

```diff
@@ -23,10 +23,10 @@ yarn-install: docker-build
 	$(DOCKER_RUN) ${DOCKER_IMAGE_TAG} yarn install --frozen-lockfile --check-files
 
 watch: yarn-install
-	$(DOCKER_RUN) --expose=8000 -p=8000:8000 -p=35729:35729 $(DOCKER_IMAGE_TAG) yarn gulp serve
+	$(DOCKER_RUN) --expose=8000 -p=8000:8000 -p=35729:35729 ${DOCKER_IMAGE_TAG} yarn gulp serve
 
 build: yarn-install
 	$(DOCKER_RUN) ${DOCKER_IMAGE_TAG} yarn gulp create-dist
 
 deploy: build
-	$(DOCKER_RUN) -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock ${DOCKER_IMAGE_TAG} rsync --no-v -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -az --delete dist/onboarder/ akeneo@$${HOSTNAME}:/home/akeneo/onboarder
+	$(DOCKER_RUN) -v /etc/passwd:/etc/passwd:ro -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock ${DOCKER_IMAGE_TAG} rsync --no-v -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -az --delete dist/onboarder/ akeneo@$${HOSTNAME}:/home/akeneo/onboarder
```

Comment:
why?

Reply:
You can ask @damien-carcel, but without that, it doesn't work

Reply:
You ask @BitOne, he is the one who did that on `pim-docs` to make the deployment work, but I don't know/remember why either!

Reply:
quite possible because we reference `akeneo` instead of the UID, so it needs to be present in the etc/passwd to be unaliased